### PR TITLE
Fix bug of Login being invisible on mobile

### PR DESCRIPTION
**Pull request by klzns**
#### What problem is this solving?

If you open https://storetheme.vtex.com/ on mobile you can see that the login component is not visible:
image

That happens because the label it's hidden when it's mobile and the `showIconProfile` default value is `undefined`.

#### How should this be manually tested?

https://breno--storecomponents.myvtex.com/

Also tested on:
- alssports
- boticario
- tacktoftheday
- samsungar
- motorolaus
- paguemenos
- ecowaterqa (improved alignment on mobile)
- babycottonsar
- biscoind (improved alignment on mobile)
- redoxx (small change https://breno--redoxx.myvtex.com/)
- umbro
- admitone
- gympassbr. Need to fix the CSS:

<table>
<thead>
<tr>
<td>Before</td><td>After</td>
</tr>
</thead>
<tbody>
<tr>
<td>

```css
.vtex-login-2-x-container button {
    background: url(https://gympassus.vteximg.com.br/arquivos/ids/155411) no-repeat 6px center;
}
```

</td>
<td>

```css
.vtex-login-2-x-container button {
    background: url(https://gympassus.vteximg.com.br/arquivos/ids/155411) no-repeat -1px center;
}
```

</td>
</tbody>
</table>
- hushpuppiescl. Need to fix mobile CSS:

<table>
<tr><td>Add</td><td>Remove</td></tr>
<tr><td>

```css
/* vtex.login.css */
.container :global(.vtex-button__label) {
    background: url(/arquivos/user-2.png) no-repeat;
    height: 23px;
    background-size: contain;
    margin-left: 1em;
    margin-bottom: 1px;
    margin-right: 1px;
}
```

</td><td>

CSS defintions that have the selectors:
- `.container .label:before, .profile:before`
- `.container .label:after, .profile:before`

</td></table>

#### Types of changes
- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Prop `showIconProfile` to be a responsive value
+- Change default of `showIconProfile` of mobile to `true`
+- Make "Sing in" text visible when `showIconProfile` is `false` and it's mobile
```
Comment (kaisermann) [inline CHANGELOG.md]:
```suggestion
- Prop `showIconProfile` to be a responsive value.
- Change default of `showIconProfile` of mobile to `true`.
- Make "Sing in" text visible when `showIconProfile` is `false` and it's mobile.
```

File: react/components/LoginComponent.js [hunk 2]
```diff
@@ -1,18 +1,18 @@
-import React, { Component, Fragment, Suspense } from 'react'
+import React, { Fragment, Suspense } from 'react'
+import { useIntl } from 'react-intl'
+import classNames from 'classnames'
+
 import {
-  withRuntimeContext,
+  useRuntime,
```
Comment (rafarubim) [inline react/components/LoginComponent.js]: ♥️

**Comment by rnbez** [comment]
@klzns @rafarubim what's the status here? Can we move on with this PR or close it?

**Comment by rnbez** [comment]
I'm closing this for inactivity.

**Pull request closed without merging**